pwndbg/dbg/__init__.py

```diff
@@ -5,14 +5,47 @@
 from __future__ import annotations
 
 from typing import Any
+from typing import Callable
+from typing import List
 from typing import Tuple
 
 dbg: Debugger = None
 
 
+class Session:
+    """
+    Interactive debugger session. Handles things like commands and history.
+    """
+
+    def history(self) -> List[str]:
+        """
+        The command history of this interactive session.
+        """
+        raise NotImplementedError()
+
+    def lex_args(self, command_line: str) -> List[str]:
+        """
+        Lexes the given command line into a list of arguments, according to the
+        conventions of the debugger being used and of the interactive session.
+        """
+        raise NotImplementedError()
+
+
+class CommandHandle:
```

Comment:
Why do we need this class? Even if the debugger supports removing commands, it makes more sense to me to have a `remove_command` method in the debugger class itself.

Reply:
It makes it cleaner for command removal under LLDB. Because of some quirks of LLDB, we need to store a global object in the root of the module, having a class makes it so that we don't have to store the name for it in a dictionary in the `Debugger` class.